### PR TITLE
Newsletter 259 translate in french

### PR DESCRIPTION
**Pull request by Copinmalin**

### CODE REVIEW

**Comment by murchandamus** [comment]
This fails because it is linking to  'specials/policy/fr/07-ressources-du-reseau.md', but that file is not here. I think it would be fixed by first merging #1182

File: _posts/fr/2023-05-17-policy.md [hunk 1]
```diff
@@ -63,7 +63,25 @@ h2:not(:first-of-type) { margin-top: 3em; }
...
+
+{% include specials/policy/fr/07-ressources-du-reseau.md %}
+
+## La politique comme interface
+
+*Originally published in [Newsletter #258](/fr/newsletters/2023/07/05/#en-attente-de-confirmation-8--la-politique-comme-interface)*
+
+{% include specials/policy/fr/08-interface.md %}
+
+## En attente de confirmation #9 : Propositions de politique
```
Comment (bitschmidty) [inline _posts/fr/2023-05-17-policy.md:78]: No prefix ("En attente de confirmation #9 :") needed for the policy page

File: _posts/fr/newsletters/2023-07-12-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+[policy series]: /fr/blog/waiting-for-confirmation/
+[news31 data loss]: /en/newsletters/2019/01/29/#fn:fn-data-loss-protect
+[news67 bolts642]: /en/newsletters/2019/10/09/#bolts-642
+[lnd v0.16.4-beta]: https://github.com/lightningnetwork/lnd/releases/tag/v0.16.4-beta
+[russell clean up]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-June/004001.html
+[review club 27625]: https://bitcoincore.reviews/27625
+[wiki getdata]: https://en.bitcoin.it/wiki/Protocol_documentation#getdata
+[news125 descriptor wallets]: /en/newsletters/2020/11/25/#how-will-the-migration-tool-from-a-bitcoin-core-legacy-wallet-to-a-descriptor-wallet-work
+[news172 descriptor wallets]: /en/newsletters/2021/10/27/#bitcoin-core-23002
+[news230 descriptor wallets]: /fr/newsletters/2022/12/14/#avec-la-dépréciation-des-anciens-portefeuilles-bitcoin-core-sera-t-il-capable-de-signer-des-messages-pour-une-adresse
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2023-07-12-newsletter.md:168]: This link does not anchor as expected (in my browser)

**Review by bitschmidty — COMMENTED** [review]
Took out a prefix on the policy series page

Corrected an anchor link (that was broken for me)

Rebased on master

**Pull request merged**